app/build.gradle

```diff
@@ -35,19 +35,25 @@ android {
             versionNameSuffix "-dev"
             resValue "string", "issuer_url", "https://phsasmarthealthcard-dev.azurewebsites.net/v1/trusted/.well-known/issuers.json"
             resValue "string", "rules_url", "https://ds9mwekyyprcy.cloudfront.net/rules.json"
+            resValue "string", "deferrals_domain", "pvc.service.yukon.ca"
+            buildConfigField 'boolean', 'IS_PROD', 'false'
         }
 
         stage {
             dimension "version"
             versionNameSuffix "-stage"
             resValue "string", "issuer_url", "https://phsasmarthealthcard-dev.azurewebsites.net/v1/trusted/.well-known/issuers.json"
             resValue "string", "rules_url", "https://phsasmarthealthcard-dev.azurewebsites.net/v1/Covid19Proof/.well-known/rules.json"
+            resValue "string", "deferrals_domain", "pvc.service.yukon.ca"
+            buildConfigField 'boolean', 'IS_PROD', 'false'
         }
 
         prod {
             dimension "version"
             resValue "string", "issuer_url", "https://smarthealthcard.phsa.ca/v1/trusted/.well-known/issuers.json"
             resValue "string", "rules_url", "https://smarthealthcard.phsa.ca/v1/Covid19Proof/.well-known/rules.json"
+            resValue "string", "deferrals_domain", "pvc.service.yukon.ca"
```

Comment:
I've seen that deferrals_domain are always the same regardless of the ProductFlavor, is there a specific reason to map it here?

Reply:
They may change, the only reason to have them here is to make it more easy to modify in the future.

Reply:
To be honest, since the url is kind of a reference just for identifying the exempt status validity, I don't really know for shure if it will remain the same in all the cases. I will further confirm that with the team and depending on the answer I will move this domain to default Config

Reply:
@bsavoini-fw, This SDK is used in multiple projects and right now we have requirements for one province.
All the provinces may follow this sooner or later following health ministry guidelines.

So making it configurable and passing the same value for a specific province makes sense.

I hope this justifies your query?

Reply:
Thanks for the detailed explanation guys! 👍